storage/src/test/java/org/apache/kafka/tiered/storage/integration/BaseDeleteSegmentsTest.java

```diff
@@ -55,7 +56,10 @@ protected void writeTestSpecifications(TieredStorageTestBuilder builder) {
                 .expectSegmentToBeOffloaded(broker0, topicA, p0, 2, new KeyValueSpec("k2", "v2"))
                 .expectEarliestLocalOffsetInLogDirectory(topicA, p0, 3L)
                 .produceWithTimestamp(topicA, p0, new KeyValueSpec("k0", "v0"), new KeyValueSpec("k1", "v1"),
-                        new KeyValueSpec("k2", "v2"), new KeyValueSpec("k3", "v3", System.currentTimeMillis()))
+                        // Using a future timestamp to test the log segment rolling mechanism.
+                        // This simulates the passage of time, triggering time-based log segment rolling conditions
+                        // When remote storage is enabled, this helps test how segments are marked for deletion
```

Comment:
This comment is not very clear to me. How about the following?

`DeleteSegmentsByRetentionTimeTest uses a really small retention time, which could cause the active segment to be rolled and deleted. We use a future timestamp to prevent that from happening.`